### ISSUE TITLE
Lines in a markdown buffer are not commented out

### ISSUE THREAD

**Issue by ichitera** [tyru/caw.vim#106]
## Issue

caw.vim do not comment out lines in a markdown buffer but creates block quote for the lines.

For example, if you call `<Plug>(caw:zeropos:toggle)` on the first line in the following markdown (a), the line becomes the markdown text (b).

**(a) Original Text:**

```markdown
I want to comment out this line.
```
**Preview of (a):**

I want to comment out this line.

**(b) Unexpected Text:**

```markdown
> I want to comment out this line.
```

**Preview of (b):**

> I want to comment out this line.

## What I expected

caw. vim surrounds lines with `<!--` and `-->`.

In the above example, I expected the following markdown text (c).

**(c) Expected Text:**

```markdown
<!-- I want to comment out this line. -->
```

**Preview of (c):**

(The line is invisible because the line is commented out.)

## The way to reproduce

1. execute `vim -u vimrc -N` (the `vimrc` is attached below)
2. execute `:e sample.md`
3. insert some text
4. type `gcc` in a line
5. _the line will get a prefix `> `_

The vimrc used in the above steps is the following:

```viml
language C
set runtimepath+=~/.vim/bundle/vim-operator-user/
set runtimepath+=~/.vim/bundle/caw.vim/
set runtimepath+=~/.vim/bundle/caw.vim/after/
filetype plugin indent on

let g:caw_operator_keymappings = 1
nmap gcc <Plug>(caw:zeropos:toggle)
```

## Environment

- OS: Windows 10 64bit
- Vim: 8.1.2200
- kana/vim-operator-user: kana/vim-operator-user@c3dfd41c1ed516b4b901c97562e644de62c367aa
- tyru/caw.vim: tyru/caw.vim@59d1bbdec7d82740811806c4d4da1963089064a8

**Comment by Inazuma110** [comment]
I also confirmed the same problem.

**Comment by tyru** [comment]
> Vim: 8.2200

You mean `Vim: 8.1.2200` ?

**Comment by ichitera** [comment]
Yes, I mean `Vim: 8.1.2200`. I made a mistake.  I'm sorry, and I corrected the mistake.

**Comment by tyru** [comment]
@ichitera @Inazuma110 Thanks for bug report. But I can't reproduce this.

What does vim output `:echo &ft` and `:verb set commentstring?`
after `2. execute :e sample.md` ?

**Comment by tyru** [comment]
![caw-issue-106](https://user-images.githubusercontent.com/48169/76669477-2022ab00-65cf-11ea-89db-ba85737b4be7.gif)

**Comment by ichitera** [comment]
Thank you for trying to reproduce the issue.
I executed `verb set commentstring?`, and I found that this issue came from [a runtime script of Vim](https://github.com/vim/vim/commit/c08ee7476b19f9b4de5df287797af87c4e3fba0a#diff-cfa570ae24a085789230fde9ace21cc2R12).
I confirmed that this issue was already solved in the latest version of Vim (version 8.2.378).


**Execution Result:**

Vim 8.1.2200:

```viml
echo &ft
" markdown

verb set commentstring?
" commentstring=> %s
"       Last set from ~C:\vim_8.1.2200\ftplugin\markdown.vim line 12
```

Vim 8.2.378:
```viml
echo &ft
" markdown

verb set commentstring?
" commentstring=<!--%s-->
"       Last set from ~C:\vim_8.2.378\ftplugin\markdown.vim line 12
```

I'm sorry to confuse you.

**Comment by tyru** [comment]
I see. thanks.
